.vscode/settings.json

```diff
@@ -0,0 +1,441 @@
+{
+    "Lua.workspace.maxPreload": 20000,
+    "Lua.diagnostics.globals": [
```

Comment:
I wonder, is there a way we could centralize global definitions for linting?  This looks like its following the luacheck CI script, but we might want something easier to maintain in the future (once this list begins to hopefully shrink)

Reply:
One difference between the CI and this is that the CI scraped luautils.cpp for the function names, so it's a little messy